go.mod

```diff
@@ -12,6 +12,15 @@ require (
 )
 
 require (
+	github.com/golang/protobuf v1.5.2 // indirect
+	golang.org/x/net v0.9.0 // indirect
+	google.golang.org/genproto v0.0.0-20200526211855-cb27e3aa2013 // indirect
+	google.golang.org/grpc v1.48.0 // indirect
+	google.golang.org/protobuf v1.28.1 // indirect
+)
+
+require (
+	github.com/PretendoNetwork/grpc-go v1.0.1
```

Comment:
Not sure how did this happen? This requirement was put in the indirect requirements and another `require ()` field was created

Reply:
VSCode decided it didn't want to play nice today. Fixed!